README.md

```diff
@@ -158,6 +158,8 @@ Docker EE is on the same code base as Docker CE, so also built from Moby, with c
 - [rocker](https://github.com/grammarly/rocker) - Extended Dockerfile builder. Supports multiple FROMs, MOUNTS, templates, etc. by [grammarly](grammarly).
 - [Stacker](https://github.com/stacker/stacker-cli) - Docker Compose Templates. Stacker provides an abstraction layer over Docker Compose and a better DX (developer experience).
 - [Zodiac](https://github.com/CenturyLinkLabs/zodiac) :skull: - A lightweight tool for easy deployment and rollback of dockerized applications. By [@CenturyLinkLabs][CenturyLinkLabs]
+- [plash](https://github.com/ihucos/plash) - A container run and build engine - runs inside docker.
```

Comment:
Every projects is order alphabetically. It should be inserted here: https://github.com/veggiemonk/awesome-docker/pull/536/files#diff-04c6e90faac2675aa89e2176d2eec7d8R157